MAINTAINERS.md

```diff
@@ -52,6 +52,8 @@ If you are an owner of the organization, you can see an automated list
   [23 Nov 2021](https://github.com/tldr-pages/tldr/issues/7404) — present
 - **258204 ([@258204](https://github.com/258204))**:
   [10 December 2021](https://github.com/tldr-pages/tldr/issues/7522) — present
+- Nicolas Hansse ([@Nico385412](https://github.com/Nico385412)):
```

Comment:
and in bold
```suggestion
- **Nicolas Hansse ([@Nico385412](https://github.com/Nico385412))**:
```

Reply:
done ! 
Sorry i didn't understood at first the time that `**....**` was for the actual maintainers 🤦